app/react/Documents/components/DocumentSidePanel.js

```diff
@@ -1,9 +1,11 @@
+/* eslint-disable max-lines */
```

Comment:
Do we want to commit this on non-spec files?  It's indispensable while working, but having a "reminder" that this is an overly long file may be useful?

Reply:
totally agree, removed.